ouroboros/shared/value.go

```diff
@@ -143,7 +143,7 @@ type Coin struct {
 }
 
 func ValueFromCoins(coins ...Coin) Value {
-	var value Value
+	value := Value{}
```

Comment:
I had to do a similar fix to get my branch to work. I'd appreciate this merged ASAP.